auth/realm/base/src/main/java/org/wildfly/security/auth/realm/FileSystemSecurityRealm.java

```diff
@@ -745,6 +879,10 @@ private Void replaceIdentityPrivileged(final LoadedIdentity newIdentity) throws
                     } catch (IOException ignored) {
                         // nothing we can do
                     }
+                    if ( this.publicKey != null ) { createDigitalSignature(this.path.toString(), this.name); }
+                    if ( this.publicKey != null && ! validateDigitalSignature(this.path.toString(), this.name)) {
```

Comment:
Same here with respect to immediate verification.

Reply:
Removed